src/luarocks/cmd/new_version.lua

```diff
@@ -118,8 +121,14 @@ local function update_source_section(out_rs, url, tag, old_ver, new_ver)
    if out_rs.source.file then
       try_replace(out_rs.source, "file", old_ver, new_ver)
    end
+
+   local old_url = out_rs.source.url
    if try_replace(out_rs.source, "url", old_ver, new_ver) then
-      return check_url_and_update_md5(out_rs)
+      local ok, err_or_changed = check_url_and_update_md5(out_rs, true)
```

Comment:
I think this is the only call-site of `check_url_and_update_md5`; I think you shouldn't change the signature of the function, but instead change what it does.

Reply:
I thought about that, but I preferred to change the signature and preserve the current behaviour to avoid ripple effects.